src/service.py

```diff
@@ -37,7 +37,7 @@ def get_model_path(self, ctx: mlservicewrapper.core.contexts.ServiceContext):
         return ctx.get_parameter_value("ModelPath", required=True)
     
     def get_model_url(self, ctx: mlservicewrapper.core.contexts.ServiceContext):
-        return ctx.get_parameter_value("ModelUrl", required=True)
+        return ctx.get_parameter_value("https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin", required=True)
```

Comment:
This line is kinda funny--notice how this is a base class for two inheritors? The language detection one is just one of them, and it overrides this function with the hard-coded URL. Because of that, this function actually doesn't get called at all for language detection, and only runs for the vectorization version.

Additionally, this change is renaming the _parameter,_ not changing the return value. So for the vectorization endpoint, it'd actually be searching for an environment variable, for example, named `SERVICE_https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin`, compared to before, where it was `SERVICE_ModelUrl`.

Suffice to say, you could verify this by adding a `print` call in there, but this line doesn't actually get called at all for language detection.